src/server/db_migrations/20200428_backfill_isFile_column.sql

```diff
@@ -0,0 +1,15 @@
+-- This migration script is to be run only after backend code
+-- has been updated to write into the isFile column. This is because
+-- the column will be made NOT NULL here.
+
+BEGIN TRANSACTION;
+
+UPDATE urls SET "isFile" = false;
```

Comment:
```suggestion
UPDATE urls SET "isFile" = false WHERE "isFile" IS NULL;
```

Reply:
This is in case someone uploaded a file between launching the new application version and running the backfill